add-chain/main.go

```diff
@@ -158,6 +161,26 @@ func entrypoint(ctx *cli.Context) error {
 		return fmt.Errorf("error generating chain config .yaml file: %w", err)
 	}
 
+	// Ensure rollup config generated by op-node is equivalent to
+	// pre-existing one used to create superchain config files
+	opNodeRollupConfig, err := rollup.LoadOPStackRollupConfig(rollupConfig.ChainID)
+	if err != nil {
+		return fmt.Errorf("failed when using op-node to generate rollup config: %w", err)
+	}
+	fmt.Printf("Attempting to read from %s\n", rollupConfigPath)
+	file, err := os.ReadFile(rollupConfigPath)
+	if err != nil {
+		return fmt.Errorf("failed to read from local rollup.json config file: %w", err)
+	}
+	var localRollupConfig *rollup.Config
+	if err = json.Unmarshal(file, &localRollupConfig); err != nil {
+		return fmt.Errorf("failed to unmarshal local rollup.json into rollup.Config struct: %w", err)
+	}
```

Comment:
There's potentially scope for DRYing this out a bit, I believe we are doing something similar to this inside of `constructChainConfig` on L148? I think the difference is that here you are unmarshalling into the op-node type instead of the superchain registry type. I suppose that makes sense if you are going to do a `reflect.DeepEqual` check.

Reply:
Since we have to move the check-rollup-config code outside of this main function (to solve import updates), this comment is no longer applicable